ansible_base/rbac/models.py

```diff
@@ -30,9 +30,13 @@
 class DABPermission(models.Model):
     "This is a minimal copy of auth.Permission for internal use"
 
-    name = models.CharField("name", max_length=255)
-    content_type = models.ForeignKey(ContentType, models.CASCADE, verbose_name="content type")
-    codename = models.CharField("codename", max_length=100)
+    name = models.CharField("name", max_length=255, help_text=_("The name of this permission."))
```

Comment:
You'll find that in a lot of these, they are not exposed in the API.

This model is also a mirror of `auth.Permission` so why doesn't it have help_text? Because the model it mirrors doesn't have help text.

The permission name is more of a python / admin pages display oriented thing. Hopefully we don't ever use it. If you wanted to get more specific about that, you could. Again, it shouldn't be outward API-facing.

Reply:
Should I take the `help_text` off these and add them to the `.help_text.ignore` file or just leave it on for now?

Reply:
The permission model, itself, maybe one day (soon-ish) be surfaced in the API in a read-only way. So it might be good to have these, although I realize I probably just have to make a PR.